src/tools/rbd/ArgumentTypes.cc

```diff
@@ -234,7 +234,7 @@ void add_create_image_options(po::options_description *opt,
 
   opt->add_options()
     (IMAGE_ORDER.c_str(), po::value<ImageOrder>(),
-     "object order [12 <= order <= 25]")
+     "object order [12 <= order <= 25] (deprecated)")
```

Comment:
I think this should be pulled out to a separated PR. Ideally, we should actually probably just remove all deprecated options from the help output and output a warning to stderr when they are used.